hub/ui-new/src/components/main/App.tsx

```diff
@@ -130,7 +130,7 @@ const App: React.FC<mainProps> = () => {
       <Page header={Header}>
         <Route exact path="/" component={BackgroundImageHeader} />
         <PageSection >
-          <Grid gutter='sm' sm={6} md={4} lg={4} xl2={1}>
+          <Grid>
```

Comment:
could you please explain why deleting this solves the scroll issue?

Reply:
It looks like 🧙‍♂️ to me 😂

Reply:
even I did't get ,how did it work :thinking: